landlab/components/landslides/landslide_probability.py

```diff
@@ -0,0 +1,535 @@
+    _var_units = {
+        'topographic__specific_contributing_area': 'm',
+        'topographic__slope': 'tan theta',
+        'soil__transmissivity': 'm2/day',
+        'soil__mode_total_cohesion': 'Pa or kg/m-s2',
+        'soil__minimum_total_cohesion': 'Pa or kg/m-s2',
+        'soil__maximum_total_cohesion': 'Pa or kg/m-s2',
+        'soil__internal_friction_angle': 'degrees',
+        'soil__density': 'kg/m3',
+        'soil__thickness': 'm',
+        'soil__mean_relative_wetness': 'None',
+        'landslide__probability_of_failure': 'None',
+        }
+
+# grid centering of each field and variable
+    _var_mapping = {
+        'topographic__specific_contributing_area': 'node',
+        'topographic__slope': 'node',
+        'soil__transmissivity': 'node',
+        'soil__mode_total_cohesion': 'node',
+        'soil__minimum_total_cohesion': 'node',
+        'soil__maximum_total_cohesion': 'node',
+        'soil__internal_friction_angle': 'node',
+        'soil__density': 'node',
+        'soil__thickness': 'node',
+        'soil__mean_relative_wetness': 'node',
+        'landslide__probability_of_failure': 'node',
+        }
+
+# short description of each field
+    _var_doc = {
+        'topographic__specific_contributing_area':
+            ('specific contributing (upslope area/cell face )' +
+             ' that drains to node'),
+        'topographic__slope':
+        'slope of surface at node represented by tan theta',
+        'soil__transmissivity':
+            ('mode rate of water transmitted' +
+             ' through a unit width of saturated soil'),
+        'soil__mode_total_cohesion':
+        'mode of combined root and soil cohesion at node',
+        'soil__minimum_total_cohesion':
+        'minimum of combined root and soil cohesion at node',
+        'soil__maximum_total_cohesion':
+        'maximum of combined root and soil cohesion at node',
+        'soil__internal_friction_angle':
+            ('critical angle just before failure' +
+             ' due to friction between particles'),
+        'soil__density': 'wet bulk density of soil',
+        'soil__thickness': 'soil depth to restrictive layer',
+        'soil__mean_relative_wetness':
+            ('Indicator of soil wetness;' +
+             ' relative depth perched water table' +
+             ' within the soil layer'),
+        'landslide__probability_of_failure':
+            ('number of times FS is <=1 out of number of' +
+             ' iterations user selected'),
+        }
+
+# Run Component
+    @use_file_name_or_kwds
+    def __init__(self, grid, number_of_iterations=250,
+                 groundwater__recharge_distribution='uniform',
+                 groundwater__recharge_min_value=20.,
+                 groundwater__recharge_max_value=120.,
+                 groundwater__recharge_mean=None,
+                 groundwater__recharge_standard_deviation=None,
+                 groundwater__recharge_HSD_inputs=[],
+                 seed=0, **kwds):
+
```

Comment:
Great! Also, I don't think there are spaces between the docstring and first line of code in PEP8.